srv/src/main/resources/application.yaml

```diff
@@ -1,3 +1,7 @@
+server.servlet.encoding:
```

Comment:
I'd move this to the same document in the multi-document YAML, where we also define the static-locations. The setting is only required together with the static-locations config and we could move that in the future to the same profile, if desired. In the cloud deployment the static resources are served by approuter and not by the Spring Boot application anymore.